matplotcheck/notebook.py

```diff
@@ -77,7 +77,13 @@ def remove_comments(input_string):
         input_string.
     """
     split_lines = input_string.split("\n")
-    return "".join([line.split("#")[0] for line in split_lines])
+    return "\n".join(
+        [
+            line.split("#")[0]
+            for line in split_lines
+            if len(line.split("#")[0]) != 0
+        ]
+    )
 
 
 def import_test(var_dict, n):
```

Comment:
i think we want to rename this function. maybe something like 
`test_package_imports` 
then we may want to clean up the docstring once we understand how it works and whether there are edge cases. i do think we NEED a notebook to test all of the things in this module.

Reply:
`test_imports_at_top`

Reply:
and maybe make the docstring more clear?